fre/make/create_compile_script.py

```diff
@@ -1,17 +1,40 @@
 '''
-TODO: make docstring
+Create a compile script to generate a model executable.
 '''
+
 import os
 import logging
-fre_logger = logging.getLogger(__name__)
 
 from pathlib import Path
 from multiprocessing.dummy import Pool
 
-from .gfdlfremake import varsfre, yamlfre, targetfre, buildBaremetal
 import fre.yamltools.combine_yamls_script as cy
+from .gfdlfremake import varsfre, yamlfre, targetfre, buildBaremetal
+
+fre_logger = logging.getLogger(__name__)
 
 def compile_create(yamlfile, platform, target, jobs, parallel, execute, verbose):
+    """Create the compile script for bare-metal build
+
+    :param yamlfile: Model compile YAML file
+    :type yamlfile: str
+    :param platform: FRE platform
+    :type platform: str
+    :param target: Predefined FRE targets
+    :type target: str
+    :param jobs: Number of jobs to run simultaneously
```

Comment:
njobs instead of jobs?

And number of compile jobs?  It's not clear what jobs are